common/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -22,5 +22,7 @@ databaseChangeLog:
       file: db/changelog/v001/update_attestation_date.sql
   - include:
       file: db/changelog/v001/add_concurrent_job_column.sql
+  - include:
```

Comment:
I think this should be moved to the end of this file, i.e. be the last change set.

Reply:
Product of a merge. I'll move it.